src/styles/less/ParentDashboard.less

```diff
@@ -71,6 +71,65 @@
   }
 }
 
+#parent-dashboard-page {
+
+
```

Comment:
Great job!  The modal looks great and everything functions well.  This is a small thing, but are you are able to make the modal background color black so that it matches the Figma?

Reply:
Cannot change the modal to black because it is a png. Need to design a png in black of same size.  Currently this doesn't exist in the Figma, so modal color will need to stay blue.